shub/deploy.py

```diff
@@ -75,7 +75,7 @@
 @click.option("-k", "--keep-log", help="keep the deploy log", is_flag=True)
 def cli(target, version, debug, egg, build_egg, verbose, keep_log):
     conf, image = load_shub_config(), None
-    if target in conf.projects:
+    if target != 'default':
```

Comment:
If that's the case I'd say it's definitely would be better to move build-egg code outside of the build cmd and remove some extra if-else's